src/tabs/tabs.scss

```diff
@@ -112,6 +111,32 @@
   }
 }
 
+@mixin iui-tabs-horizontal {
+  flex-direction: column;
+
+  .iui-tabs {
+    display: flex;
+  }
+}
+
+@mixin iui-tabs-vertical {
+  flex-direction: row;
+
+  .iui-tabs {
+    display: inline-flex;
+    flex-direction: column;
+
+    li,
+    .iui-tab {
+      width: 100%;
+    }
+
+    ~ .iui-tabs-content {
+      flex-grow: 1;
```

Comment:
Is it needed?

Reply:
Without this, if there is little within the content area it does not take the full parent's width.  To demonstrate I've turned the content area pink.

Without this CSS:
![Screen Shot 2021-08-17 at 12 53 32 PM](https://user-images.githubusercontent.com/849817/129768354-f79f006a-559b-45ab-896e-132dd5e0dda5.png)

With this CSS:
![Screen Shot 2021-08-17 at 12 55 03 PM](https://user-images.githubusercontent.com/849817/129768427-5f5d7bc3-92c7-4ee0-aa5d-a317abfb41d8.png)